src/main/java/com/microsoft/aad/adal4j/AdalAuthorizatonGrant.java

```diff
@@ -62,6 +62,12 @@
         if (this.params != null) {
             outParams.putAll(this.params);
         }
+        
+        if(!outParams.containsKey("scope"))
+        {
+        	outParams.put("scope", "openid");

```

Comment:
I think the important thing is that there are no extra QP allowed in the username password overloads (token endpoint is a protocol artifact that is involved regardless of the overload).  

Sorry, I can't help being pedantic.  

You're suggestion sounds great.  Please proceed.